openedx/adg/lms/applications/tests/constants.py

```diff
@@ -0,0 +1,7 @@
+"""
+Constants for all the tests.
+"""
+
+USERNAME = 'test'
+EMAIL = 'test@example.com'
+PASSWORD = 'edx'
```

Comment:
Both the test classes have to be `setUp()` differently as I've explained in related comments. 
I'd prefer to keep these constants in a separate file. However, if we HAVE to avoid a separate file, we could instead opt to make these global constants in the _test_views.py_ file.